python/python/glide/async_commands/standalone_commands.py

```diff
@@ -665,3 +665,72 @@ async def random_key(self) -> Optional[str]:
             Optional[str],
             await self._execute_command(RequestType.RandomKey, []),
         )
+
+    async def scan(
+        self,
+        cursor: int,
```

Comment:
The cursor should be a str to be consistent with other SCAN commands. We used a str for other ones because the cursor is a 64-bit unsigned integer.